src/Ilios/CoreBundle/Entity/Manager/PermissionManagerInterface.php

```diff
@@ -91,6 +91,14 @@ public function userHasReadPermissionToProgram(UserInterface $user, ProgramInter
      * @return bool
      */
     public function userHasReadPermissionToSchool(UserInterface $user, SchoolInterface $school);
+     
+    /**
+    * Checks if a given user has "write" permissions for a list of schools
+    * @param UserInterface $user
+    * @param ArrayCollection[SchoolInterface] $schools
+    * @return bool
+    */
+    public function userHasWritePermissionToSchools(UserInterface $user, ArrayCollection $schools);
```

Comment:
interface should also declare `public function userHasReadPermissionToSchools(...)`.